src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -40,7 +40,7 @@ import "./ha-more-info-logbook";
 import "./controls/more-info-default";
 
 const DOMAINS_NO_INFO = ["camera", "configurator"];
-const EDITABLE_DOMAINS_WITH_ID = ["scene", "automation"];
+const EDITABLE_DOMAINS_WITH_ID = ["scene", "automation", "person"];
```

Comment:
The person will always have an ID, also when it is not editable (YAML).

Reply:
There is an attribute, `editable: true` that we should use.

Reply:
Ah, interesting. I'll update to handle that case.

Would you be alright with it if I extracted this conditional from the HTML template into a getter method so that it's a bit easier to work with?
https://github.com/home-assistant/frontend/blob/34c2742d88da405293671dcf3c1c17a6e85be5fa/src/dialogs/more-info/ha-more-info-dialog.ts#L140-L143

Reply:
Yeah that's fine, I would not add `person` to the `EDITABLE_DOMAINS_WITH_ID` though, as it would need a different check.

Reply:
I think I understand a bit better now -- I understood `EDITABLE_DOMAINS_WITH_ID` to mean "domains that are editable, and have an ID parameter that should be passed along", but it seems to instead mean "domains that are only editable if an ID is present". I may add a JSDoc comment to that effect, if that's fine with you.